docs/resource-mapper.md

```diff
@@ -0,0 +1,20 @@
+# OpenTelemetry Resource mapper
+
+Turn the resource mapper in the metric exporter into a shared tool. It can be reused for both trace and metric exporters. 
+
+## Motivation
+
+OpenTelemetry Resource labels now can be inserted into trace spans. We want to allow Monitored
+Resources to be inserted as well. A shared resource mapper can enable trace exporter to convert
```

Comment:
"We want to allow Monitored Resources **(as derived from OpenTelemetry resources via `resourceToMonitoredResourcepb`)** to be inserted as well **(as well or instead?)**."